src/main.js

```diff
@@ -212,6 +217,8 @@ class RoundSlider extends LitElement {
   }
 
   firstUpdated() {
+    if(!this._enabled || this.disabled) return;
+
```

Comment:
With this, the `disabled` property cannot be removed.

Reply:
Im not sure I understand what you mean by cannot be removed

Reply:
```js
constructor() {
  this.disabled = true;
}
firstUpdated() {
  this.disabled = false;
}
render() {
<round-slider
?disabled=${this.disabled}
></round-slider>
}
```

The event listeners will never be registered, and the slider won't work.

Reply:
Ah gotcha. Good catch. Added this to the updated. Also update the `_enabled` name more convey what it is really doing